### ISSUE TITLE
please add 営業時間

### ISSUE THREAD

**Issue by mcfiredrill** [osscafe/osscafe.net#7]
Can you add the hours you are open? I didn't see it anywhere on the site.

**Comment by cognitom** [comment]
Exactly. We've missed the information about it on our English page. Thanks!

**Comment by mcfiredrill** [comment]
Ah I see, its on the Japanese side! Well at least I found it. ><
